docs/api/webob.txt

```diff
@@ -23,85 +23,71 @@ a certain type:
 The classes that may be returned by one of the functions above, and their
 methods:
 
-.. autoclass:: Accept
-   :members: parse
-
 .. autoclass:: AcceptOffer
    :members: __str__
 
-.. autoclass:: AcceptValidHeader
-   :members: parse, header_value, parsed, __init__, __add__, __bool__,
-	     __contains__, __iter__, __radd__, __repr__, __str__,
-	     accept_html, accepts_html, acceptable_offers, best_match, quality
-
-.. autoclass:: AcceptNoHeader
-   :members: parse, header_value, parsed, __init__, __add__, __bool__,
-	     __contains__, __iter__, __radd__, __repr__, __str__,
-             accept_html, accepts_html, acceptable_offers, best_match, quality
+.. autoenum:: HeaderState
```

Comment:
"and their methods" makes the sentence quite ambiguous, and as the documentation for the classes also includes their non-method attributes, perhaps "and their methods" could be removed? (Assuming I'm reading the intended meaning correctly.)

Under this, `AcceptOffer` and `HeaderState` are included - but they are not classes that "may be returned by one of the functions above"?